### DISCUSSION TITLE
Use pipx instead of pip

### DISCUSSION THREAD

**Discussion by ithinkido** [svg2mod/svg2mod#45] [Ideas]
It would be great to use pipx for installing this package. In short - pipx is pip + virtual environment +path management  all rolled in to one with any need on the user side to worry about anything. The package becomes available globally, and will not be effected by changes to the python versions that so notoriously break dependencies. For more info - https://github.com/pypa/pipx

This PR updates the install instructions to use pipx.  https://github.com/svg2mod/svg2mod/pull/44

**Comment by Sodium-Hydrogen** [comment]
With multiple options available for managing python virtual environments I will admit I did not know about pipx until now.

I do find it to be a useful tool in general. While it may not pose any significant benefit to svg2mod, with the single dependency it has, I think a shorter note on the advantages of pipx would be better suited and to list both the traditional and safe installations.

I think the concept of a tool to automatically enforce best practice is inherently a good idea. I, however, will not use pipx as I already have my own workflow. Also since not all svg2mod users are python developers who frequently install packages, less tools to learn and use are more likely to be a benefit to them. This is why I would like both to be in the readme.
